packages/eslint-plugin/docs/rules/consistent-type-assertions.md

```diff
@@ -12,15 +12,18 @@ Type assertions are also commonly referred as "type casting" in TypeScript (even
 type Options =
   | {
       assertionStyle: 'as' | 'angle-bracket';
-      objectLiteralTypeAssertions: 'allow' | 'allow-as-parameter' | 'never';
+      objectLiteralTypeAssertions?: 'allow' | 'allow-as-parameter' | 'never';
+      allowAsReturn?: boolean;
```

Comment:
this shouldn't be a separate boolean flag.
It would be better to accept an array of strings, i.e.:
```ts
type Options = 
  | {
    assertionStyle: 'as' | 'angle-bracket';
    objectLiteralTypeAssertions?:
      | 'allow'
      | 'allow-as-parameter'
      | 'allow-as-return'
      | 'never'
      | ('allow-as-parameter' | 'allow-as-return')[];
  }
  | {
    assertionStyle: 'never';
  }
```